spec/spec_helper.rb

```diff
@@ -1,8 +1,12 @@
-# Add simplecov
-require 'minitest'
-require 'minitest/autorun'
-require 'minitest/reporters'
+require "simplecov"
+SimpleCov.start
+require "minitest"
+require "minitest/autorun"
+require "minitest/reporters"
 
 Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
 
 # Require_relative your lib files here!
+require_relative "../lib/Reservation"
+require_relative "../lib/Hotel"
+require_relative "../lib/Rooms"
```

Comment:
the `Rooms` file doesn't exist`.  

Also typically in Ruby we don't capitalize file names.  Just a style thingy.